doc/architecture/sensor_view_configuration.adoc

```diff
@@ -0,0 +1,15 @@
+= Sensor view configuration
+
+The sensor view configuration configures the sensor view.
+There are two sources for sensor-view-configuration data.
+
+Sensor-view-configuration data can be provided by the environment simulation.
+In this case, the data is intended for the automatic configuration of an environment simulation in order to supply the necessary input to a sensor model.
+Sensor-view-configuration data is not intended to provide parameters to a generic sensor model. 
+
+Sensor-view-configuration data can also be provided by the sensor model to the environment simulation.
+In this case, the data describes the input configuration that is desired by the sensor model.
+In response, the environment simulation will configure the input and provide a new message which describes the actual configuration.
```

Comment:
```suggestion
In response, the simulation environment will reply with a new sensor-view-configuration message that describes the possible configuration.
```

Reply:
CCB Meeting 04.08.2021:
- wording "environment simulation" should be defined somewhere. --> Part that simulates the environement.
- "simulation environment" is not the correct wording here. "environment simulation" should be used here.

Reply:
@kmeids : Apart from "environment simulation" vs "simulation environment", your suggestion also contains some major changes:

- The environment does not configure the input.
- The message does not describe the actual configuration but possible configurations.

 Should I still apply these changes? Or is this moot since we talk about  "environment simulations" and not "simulation environment"? I hope I am not confusing anything here.

Reply:
@stefancyliax
- The word "environment simulation" is mentioned here: https://github.com/OpenSimulationInterface/open-simulation-interface/blob/migration/doc/images/osi-context.png.
- This image is used here: https://github.com/OpenSimulationInterface/open-simulation-interface/blob/migration/doc/architecture/architecture_overview.adoc

I recommend to add a sentence in this topic. What do you think?

Reply:
CCB: @pmai will reorder sentences and do slight rewrite to address these comments.

Reply:
@pmai is this done?

Reply:
OSI CCB: 
- @pasched Please review the new chapter from pmai.